packages/host/app/components/modal-container.gts

```diff
@@ -113,6 +149,7 @@ export default class ModalContainer extends Component<Signature> {
       }
 
       .dialog-box__footer {
+        grid-area: footer;
```

Comment:
This extends `ModalContainer` to support a sidebar as called for in the design:

<img width="1064" alt="Boxel 2023-12-15 12-29-46" src="https://github.com/cardstack/boxel/assets/43280/63db2c12-8430-45a9-8780-5c46b80f8d65">

Is this worthy of extraction into `boxel-ui`? It’s used in a few places now.